dashboard.js

```diff
@@ -0,0 +1,16 @@
+function renderRows(branch) {
+    var liveLink = 'http://' + branch + '.dashboard/';
+    var githubBranchSrc = 'https://github.com/lowsky/dashboard/tree/' + branch;
+    return <tr className="warning" key={branch}>
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Expected an assignment or function call and instead saw an expression.](https://www.codacy.com/app/skylab71/dashboard/file/638356014/issues/source?bid=2231116&fileBranchId=2312187#l4)